shell/platform/android/io/flutter/plugin/platform/PlatformViewsResolver.java

```diff
@@ -0,0 +1,15 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugin.platform;
+
+import android.view.View;
+
+public interface PlatformViewsResolver {
```

Comment:
I think the style guide requires class level javadocs, too.

Reply:
deleted this interface